src/Sfa.Tl.Matching.Application/Services/ReferralService.cs

```diff
@@ -39,38 +41,90 @@ public class ReferralService : IReferralService
+
+
+            // await _referralService.SendProviderReferralEmail(id);
+            //This need to happen before we redirect to Email Sent Screen i.e. in the post of CheckAnswers
+            //Also you might want to move this call to Opportunity service
+
+            //Use GetNumberOfPlacements everywhere
+
+            var emailTemplate = await GetEmailTemplate(EmployerReferralEmailTemplateName);
+
+            var referrals = await GetEmployerReferrals(opportunityId);
+
+            var employer = referrals.FirstOrDefault();
+
+            if (employer == null)
+            {
+                return;
+            }
+
+            var toAddress = employer.EmployerContactEmail;
+
+            var numberOfPlacements = GetNumberOfPlacements(employer.PlacementsKnown, employer.Placements);
+
+            var tokens = new Dictionary<string, string>
             {
-                new EmailPlaceholderDto {Key = "providers_list", Value = sb.ToString()}
+                { "employer_contact_name", employer.EmployerContact },
+                { "employer_business_name", employer.EmployerName },
+                { "employer_contact_number", employer.EmployerContactPhone },
+                { "employer_contact_email", employer.EmployerContactEmail },
+                { "employer_postcode", employer.Postcode },
+                { "number_of_placements", numberOfPlacements },
+                { "route", employer.RouteName },
+                { "job_role", employer.JobTitle }
             };
-            */
+
+            var sb = new StringBuilder();
+
+            foreach (var referral in referrals)
+            {
+                sb.AppendLine($"# {referral.ProviderName}");
+                sb.AppendLine($"{referral.ProviderVenuePostcode}");
+                sb.AppendLine($"Contact name: {referral.ProviderPrimaryContact}");
+                //sb.AppendLine($"Telephone: {referral.ProviderPrimaryContactPhone}");
+                sb.AppendLine($"Email: {referral.ProviderPrimaryContactEmail}");
+                sb.AppendLine("Has students learning: ");
+
+                foreach (var qualificationShortTitle in referral.QualificationShortTitles)
+                {
+                    sb.AppendLine($"* {qualificationShortTitle}");
+                }
+                sb.AppendLine(""); //Need a blank line, otherwise the next heading won't be formatted
+            }
+
+            tokens.Add("providers_list", sb.ToString());
+
+            await _emailService.SendEmail(emailTemplate.TemplateName,
+                toAddress,
+                "Industry Placement Matching Referral",
+                tokens,
+                "mike.wild@digital.education.gov.uk");
+
+            await SaveEmailHistory(emailTemplate, tokens, opportunityId, toAddress, employer.CreatedBy);
+        }
+
+        private static string GetNumberOfPlacements(bool? placementsKnown, int? placements)
+        {
+            return placementsKnown.GetValueOrDefault()
```

Comment:
We have the same logic in the UI as this...so might be worth moving to a common place